Modules/SubModule_Import/xASL_imp_NII2BIDS_Subject.m

```diff
@@ -16,68 +16,62 @@ function xASL_imp_NII2BIDS_Subject(imPar, bidsPar, studyPar, nameSubject)
 % DESCRIPTION: Run NII to ASL-BIDS for one individual subject.
 %
 % 1. Initialize
-% 2. Process all the anatomical files
-% 3. Process the perfusion files (iterate over sessions)
+% 2. Process the anat & perfusion files
 % - 1. Make a subject directory
-% - 2. Iterate over runs
+% - 2. Iterate over sessions
+% - 3. Iterate over runs
 % 
 % -----------------------------------------------------------------------------------------------------------------------------------------------------
 % EXAMPLE:     xASL_imp_NII2BIDS_Subject(imPar, bidsPar, studyPar, nameSubject);
 % __________________________________
 % Copyright 2015-2021 ExploreASL
 
     %% 1. Initialize
+    bidsLabel = xASL_imp_CheckForAliasInVisit(imPar,nameSubjectSession);
     
-    subjectLabel = xASL_adm_CorrectName(nameSubject,2);
+    % Make a subject directory
+    if ~exist(fullfile(imPar.BidsRoot,['sub-' bidsLabel.subject]),'dir')
+        xASL_adm_CreateDir(fullfile(imPar.BidsRoot,['sub-' bidsLabel.subject]));
+    end
+    
+    %% 2. Process the anat & perfusion files
+    listSessions = xASL_adm_GetFileList(fullfile(imPar.TempRoot,nameSubjectSession),'^ASL.+$',false,[],true);
```

Comment:
Are we not mixing up sessions (visits) and runs (sessions) here? A list of sessions should in legacy be based on `_1 ` `_2` `_3` etc., not on `ASL.+`

Reply:
Yeah, this is a bit mixed up almost everywhere. It would be a lot easier if we had written down both notations for each function/section. This should be legacy visits I think (which is BIDS sessions) and there we have ASL_1 for visit 1 etc., right? I mean... it worked so far 😆

Reply:
Nope, in legacy we have sessions (BIDS runs) ASL_1 ASL_2 ASL_3.
Also, in legacy we have visits (BIDS sessions) sub-001_1 sub-001_2 sub-001_3.

Reply:
Yes. Agree. This is mixed up. But only the terminology is mixed up. It functions correctly...
We have used here in BIDS function the Legacy terminology. It is indeed incorrect to call it sessions, but it is treating it as runs, so we only need to change the names. I would do that in issue #775 - we already mention it there.